README.md

```diff
@@ -120,55 +120,32 @@ you can set the configuration variable `skipUpdateWhenPageNotFound` to `true` (d
 
 ## Command-line Autocompletion
 
-Currently we only support command-line autocompletion for zsh
-and bash. Pull requests for other shells are most welcome!
+We currently support command-line autocompletion for zsh and bash.  
+Pull requests for other shells are most welcome!
 
-### zsh
-
-It's easiest for
-[oh-my-zsh](https://github.com/robbyrussell/oh-my-zsh)
-users, so let's start with that.
-
-```zsh
-mkdir -p $ZSH_CUSTOM/plugins/tldr
-ln -s bin/completion/zsh/_tldr $ZSH_CUSTOM/plugins/tldr/_tldr
-```
-
-Then add tldr to your oh-my-zsh plugins,
-usually defined in `~/.zshrc`,
-resulting in something looking like this:
-
-```zsh
-plugins=(git tmux tldr)
-```
+To enable autocompletion for the tldr command, run:
 
-Alternatively, using [zplug](https://github.com/zplug/zplug)
+### zsh
 
 ```zsh
-zplug "tldr-pages/tldr-node-client", use:bin/completion/zsh
+tldr completion zsh
+source ~/.zshrc
 ```
 
-Fret not regular zsh user!
-Copy or symlink `bin/completion/zsh/_tldr` to
-`my/completions/_tldr`
-(note the filename).
-Then add the containing directory to your fpath:
+### bash
 
-```zsh
-fpath=(my/completions $fpath)
+```bash
+tldr completion bash
+source ~/.bashrc
 ```
 
-### Bash
+This command will generate the appropriate completion script and append it to your shell's configuration file (`.zshrc` or `.bashrc`).
 
-```bash
-ln -s bin/completion/bash/tldr ~/.tldr-completion.bash
-```
+After running the completion installation command, restart your shell or reload your configuration file to enable the autocompletion.
```

Comment:
Is this sentence really necessary, given the commands above already reloads the config file?

Reply:
Thank you for your review! :smile: 

That’s correct. Since `source ~/.bashrc` was already mentioned, I don’t think there’s a need to remind it again.

I have updated the README!